nominal/core/dataset.py

```diff
@@ -81,6 +82,8 @@ def logical_series(self) -> timeseries_logicalseries.LogicalSeriesService: ...
         def upload(self) -> upload_api.UploadService: ...
         @property
         def units(self) -> scout.UnitsService: ...
+        @property
+        def compute(self) -> scout_compute_api.ComputeService: ...
```

Comment:
ah, this type should inherit from `Channel._Clients` and then it would propagate automatically

Reply:
Runs does this as well like so:

```py
class _Clients(Dataset._Clients, Attachment._Clients, HasAuthHeader, Protocol):
```

note that you need to put the inheritance _before_ `HasAuthHeader`